book/website/collaboration/academic-industry/academic-industry-personal-story.md

```diff
@@ -0,0 +1,48 @@
+(cl-academic-industry-personal-story)=  
+# Turing-Roche Community Activities Personal Story
+
+## Introduction
+My name is [Vicky Hellon](https://www.turing.ac.uk/people/vicky-hellon) and I am the Community Manager for the Turing-Roche Strategic Partnership. 
+This 5-year partnership is developing new data science methods to investigate large, complex, clinical and healthcare datasets to better understand how and why patients respond differently to treatment, and how treatment can be improved.
+
+I started as Community Manager for the partnership in October 2021, with my previous background in scientific publishing and health data science. 
+This personal story covers some of the communuity activities and spaces I and the team have set up for the partnership which we hope will be helpful to others working in similar collaborations.
+
+### Slack Workspace 
+Our Turing-Roche Slack Workspace is one of the first spaces we set up for our community to connect.
+After we ran an initial workshop we wanted somewhere for participants to continue connecting that was easy to use, allowed asynchronous communication and a low barrier to join.
+Slack suited this purpose and we have since grown its use as somewhere to share partnership news, ask for advice and share interesting papers, jobs and opportunities, as well as its original purpose of allowing people to connect after events we have run. 
+We also have experimented with coffee chats.
+
+A learning I think that is useful to pass on is considering a strategy for growth of your Slack Workspace or other messaging/social media space. 
+The Turing-Roche Slack grew larger than we had expected, meaning we had implement things like an onboarding message and new channels retrospectively rather than considering these needs at the start.
```

Comment:
```suggestion
The Turing-Roche Slack grew larger than we had expected, which meant that we had to implement things like an onboarding message and new channels retrospectively rather than considering these needs at the start.
```